### DISCUSSION TITLE
日本Hololive旗下虛擬偶像殺入YouTube

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#266] [閒聊八卦]
VTuber（Virtual YouTuber）是指以 YouTube 爲平台進行影片直播的虛擬偶像，他們通常以虛擬形像在網路進行影片創作或直播等娛樂活動，而其收入除了靠 YouTube 內建廣告分紅以及第三方廣告外，其餘收入均由觀眾以直播時用戶付費發出的「超級即時通訊」（super chat）方式，直接付錢給 VTuber。
而近日專門收集以及統計 YouTube 數據的網站 Playboard 發布最新數據，單以 Superchat 收入計算，為 2020 世界 YouTube 虛擬偶像收入作出排名，除以遊戲實況為主的 bintroll 及 M.S.S Project 外，排行前 10 名中有 8 名都是日系偶像，且全數為是 Hoholive 旗下藝人。而奪得第一名的是會長桐生可可，其 YouTube 粉絲超過 88 萬人，獲贊助金額高達近 1.5 億日圓（約台幣 4086.7 萬元）。
[Hololive旗下的偶像已在YouTube平台上賺取近800萬美元打賞](https://fortuneinsight.com/web/feature/718837/718837-autosave-v1/)

**Comment by chun1203** [comment]
沒想到連虛擬偶像也來當youtuberXD

**Comment by wax75397** [comment]
這是多久以前的訊息阿?我的推訂閱都快破400萬了

**Comment by syt-110** [comment]
華生很推欸

**Comment by 107602505** [comment]
講到日本的虛擬偶像就會想到年收入也超高的初音未來，且動漫文化也是紅遍全球真的很佩服日本人在這些地方的設計和用心。

**Comment by mesakiyuta** [comment]
今後Vtuber行業的技術門檻會越來越低，或許在不久的未來會跟youtuber一樣，成為更加普及的職業。

除了Hololive以外，Nijisanji也好看，大推，有興趣的人可以去看看潤滑液冰壺大賽！

**Comment by cody910120** [comment]
那會有Vtuber惡龍咆嘯嗎?

**Comment by Isongzhe** [comment]
啊我還以為holo 旗下的是藝人，原來是偶像阿

**Comment by neo910831** [comment]
不得不說宅宅的消費力是真的很高，常看到一整條最貴的紅sc XD

**Comment by YuHengChen54** [comment]
日本這方面的產業真的太強了，但我完全看不懂0.0

**Comment by 109302013evantsao** [comment]
修但幾哩，你讓這個地方變得油爆了，我作夢都沒想到，會在這地方看到Vtuber這個詞

**Comment by bloodtrail0511** [comment]
Yagoo一生推 :)

**Comment by pekoranoinu** [comment]
最近虛擬偶像真的是很火熱。對於宅宅來說，虛擬偶像和真人相比，當然是二次元比較具吸引力吧~

**Comment by sean1325418** [comment]
對Vtuber的辨識技術比較有興趣，用影像辨識出目前表情(動作)，再讓虛擬人物做出同樣的表情(動作)，感覺不會到很難，說不定大學生就能做出來。

**Comment by ghost** [comment]
嘔嘔嘔 好油喔

是說holo我單推YMD>///<婆爆了

**Comment by chupochun** [comment]
日本在這種特別的方面發展，真的是讓所有人大開眼界。

**Comment by xkllkx** [comment]
最後Vtuber氾濫之後，最後吸引人的還也會是中之人。

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，讓我大開眼界。
